enterprise_subsidy/apps/subsidy/models.py

```diff
@@ -83,6 +97,7 @@ class Meta:
 
     active_datetime = models.DateTimeField(null=True, default=None)
     expiration_datetime = models.DateTimeField(null=True, default=None)
+    history = HistoricalRecords()
```

Comment:
Here's a good example of where we _do_ want history on an abstract parent model - all subsidies should track history.  But this means we can remove the `history` field from the inheriting child models below.